test/unit/lodash.spec.js

```diff
@@ -0,0 +1,690 @@
+                var countFloat = 0;
+                for (var i = 0; i < arr.length; i++) {
+                    if (typeof arr[i] !== 'number') {
+                        expect(typeof arr[i]).to.eql('number');
+                    }
+                    if (arr[i] === parseInt(arr[i])) {
+                        countInt++;
+                    } else if (arr[i] === parseFloat(arr[i])) {
+                        countFloat++;
+                    }
+                }
+
+                expect(arr.length).to.eql(8);
+                expect(countInt).to.eql(5);
+                expect(countFloat).to.eql(3);
+            });
+            it('should count repeating numbers', function () {
+                expect(_.countBy.apply(_, datasets.countBy())).to.eql({1: 1, 2: 2, 6: 2, 7: 3});
+            });
+        });
+
+        describe('countBy2 function', function () {
+            var params = datasets.countBy2();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = typeof params[1];
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should contain array of string with different strings length', function () {
+                var arr = params[0];
+                var string0 = 0;
+                var string1 = 0;
+                var string2 = 0;
+                for (var i = 0; i < arr.length; i++) {
+                    if (typeof arr[i] !== 'string') {
+                        expect(typeof arr[i]).to.eql('string');
+                    }
+                    if (arr[i].length === 3) {
+                        string0++;
+                    } else if (arr[i].length === 4) {
+                        string1++;
+                    } else if (arr[i].length === 5) {
+                        string2++;
+                    }
+                }
+
+                expect(arr.length).to.eql(6);
+                expect(string0).to.eql(2);
+                expect(string1).to.eql(3);
+                expect(string2).to.eql(1);
+            });
+            it('should count length of given strings', function () {
+                expect(_.countBy.apply(_, datasets.countBy2())).to.eql({3: 2, 4: 3, 5: 1});
+            });
+        });
+    });
+
+    describe('every', function () {
+        describe('every function', function () {
+            var params = datasets.every();
+
+            it('should match types of passing elements', function () {
+                var element1 = params[0] instanceof Array;
+                var element2 = typeof params[1];
+
+                expect(element1).to.eql(true);
+                expect(element2).to.eql('string');
+                expect(params.length).to.eql(2);
+            });
+            it('should contain array of objects', function () {
```

Comment:
Test case should indicate that we need objects with two properties if it is required.